### PR TITLE
Video player production asset links

### PR DESCRIPTION
**Pull request by JoshBowdenConcepts**
## Summary

Updating the asset location and link so that the prefixes in Gatsby

## List of notable changes:

- **changed** the location of the static assets to an "assets" folder within the "static" folder
- **updated** the URL for the assets to the new location prefixed with "brand"
- **changed** local deployment of video not to work as expected while production documentation should.

## What should reviewers focus on?

- This should not work locally but should fix the URLs in production

## Contributor checklist:

- [ ] All new and existing CI checks pass
- [ ] Tests prove that the feature works and covers both happy and unhappy paths
- [ ] Any drop in coverage, breaking changes or regressions have been documented above
- [ ] New visual snapshots have been generated / updated for any UI changes
- [ ] All developer debugging and non-functional logging has been removed
- [ ] Related issues have been referenced in the PR description

## Reviewer checklist:

- [ ] Check that pull request and proposed changes adhere to our [contribution guidelines](../../CONTRIBUTING.md) and [code of conduct](../../CODE_OF_CONDUCT.md)
- [ ] Check that tests prove the feature works and covers both happy and unhappy paths
- [ ] Check that there aren't other open Pull Requests for the same update/change

### CODE REVIEW

File: apps/docs/content/components/VideoPlayer.mdx [hunk 1]
```diff
@@ -14,30 +14,35 @@ import {VideoPlayer} from '@primer/react-brand'
 
 ## Examples
 
+{/* Note: These assets will not load in local development. The way gatsby works prefixes the assets URLs which cannot currently be done in these files. */}
```
Comment (rezrah) [inline apps/docs/content/components/VideoPlayer.mdx:17]: This renders actual markup in the docs 👇 , can you use a valid markdown comment here instead please?

Screenshot 2023-08-29 at 09 34 45

Comment (JoshBowdenConcepts) [reply apps/docs/content/components/VideoPlayer.mdx]: This has been updated in the following [SHA](https://github.com/primer/brand/pull/407/commits/4336fa1b489463d256e5d2e3aab7c83cdc0c821d) I will test the build once it renders. @rezrah what are the steps to make sure this affects prod correctly?

Comment (rezrah) [reply apps/docs/content/components/VideoPlayer.mdx]: > @rezrah what are the steps to make sure this affects prod correctly? 

Once merged, the video player docs examples should start working.

There's no way to test this on preview urls, so you can either test test in prod **or** you locally by building the prod docs and serving output on a local web serving with a `basePath` of `brand`, to simulate the prod enironment. That's how I've typically done it before.

File: apps/docs/content/components/VideoPlayer.mdx [hunk 2]
```diff
@@ -14,30 +14,37 @@ import {VideoPlayer} from '@primer/react-brand'
 
 ## Examples
 
+<!---
+ Note: These assets will not load in local development. The way gatsby works prefixes the assets URLs which cannot currently be done in these files.
```
Comment (rezrah) [inline apps/docs/content/components/VideoPlayer.mdx:18]: Technically can work in local dev environment, if you build and serve the docs with a baseURL. Here's a suggestion to explain why we're doing this.

```suggestion
TODO: Fix asset loading in environments that don't use `pathPrefix`. This is a temporary solution to load assets in production deployments, but currently doesn't work in dev mode or preview URLs
```

**Review by rezrah — APPROVED** [review]

**Pull request merged**